includes/Modules/Analytics/Advanced_Tracking.php

```diff
@@ -79,14 +97,25 @@ public function register() {
 			function() {
 				$this->set_up_advanced_tracking();
 			},
-			11
+			15
```

Comment:
What was the reason to change this priority?

Reply:
It was just to be safe. We don't need to go early and we can afford to go late. We made all actions priority 15.